app/src/main/java/org/oppia/android/app/application/ApplicationComponent.kt

```diff
@@ -99,6 +99,7 @@ import javax.inject.Singleton
     LocaleProdModule::class, ActivityRecreatorProdModule::class,
     NumericExpressionInputModule::class, AlgebraicExpressionInputModule::class,
     MathEquationInputModule::class, SplitScreenInteractionModule::class,
+    LocaleProdModule::class, ActivityRecreatorProdModule::class,
```

Comment:
This appears duplicated with a few lines above. Please remove.

Reply:
done